### PR TITLE
Group duplicates members in widget #2754

### PR DESCRIPTION
**Pull request by leonardohof**
Resolve: opencollective/opencollective#2754

### CODE REVIEW

File: components/MembershipsWithData.js [hunk 1]
```diff
@@ -91,8 +101,8 @@ class MembershipsWithData extends React.Component {
         </style>
 
         <div className="Collectives cardsList">
-          {memberships.map(membership => (
-            <Membership key={membership.id} membership={membership} LoggedInUser={LoggedInUser} />
+          {collectiveIds.map(id => (
```
Comment (Betree) [inline components/MembershipsWithData.js]: Is the goal of `collectiveIds` to preserve the original order?

Comment (leonardohof) [reply components/MembershipsWithData.js]: Yes, it is.

Comment (Betree) [reply components/MembershipsWithData.js]: Well thought :+1:

**Comment by leonardohof** [comment]
@Betree the requested changes were made :+1:

File: components/CollectiveCard.js [hunk 2]
```diff
@@ -12,38 +12,72 @@ class CollectiveCard extends React.Component {
...
-            collective.type === 'ORGANIZATION' ? (
-              <FormattedMessage id="tier.name.sponsor" defaultMessage="sponsor" />
-            ) : (
-              <FormattedMessage id="tier.name.backer" defaultMessage="backer" />
-            );
-          break;
+    const { intl, collective, membership } = this.props;
+    let { memberships } = this.props;
+    memberships = memberships || (membership ? [membership] : []);
+    console.log(memberships);
```
Comment (Betree) [inline components/CollectiveCard.js]:
```suggestion
```

**Comment by leonardohof** [comment]
> Just spotted a regression:
> 
> image
> 
> See:
> 
> * https://opencollective-frontend-vfyiwfiipt.now.sh/search?q=drupal
> * https://staging.opencollective.com/search?q=drupal

Fixed!
image

**Pull request merged**